howto/aar/configure.md

```diff
@@ -106,13 +106,21 @@ Use "aar trust [command] --help" for more information about a command.
 
 Every AMS instance has a registry-specific client certificate that is stored at `/var/snap/ams/common/registry/client.crt`. To manually register an AMS client, you'll need to copy this certificate to the machine hosting AAR and use the CLI to trust it.
 
-Use any of the following commands to do that:
+To add an AMS client as trusted [publisher](https://discourse.ubuntu.com/t/application-registry/17761#aar-roles), use the following command:
```

Comment:
Shouldn't it be
```suggestion
To add an AMS client as a trusted [publisher](https://discourse.ubuntu.com/t/application-registry/17761#aar-roles), use the following command:
```

Reply:
That's the thing I always do wrong ... yes, it should be. :) Thanks!